src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
 // write code here
+
+const listaUl = document.querySelector('ul');
+const listaLi = listaUl.querySelectorAll('li');
+
+const posortowanaLista = [...listaLi].sort((a, b) => {
+  const salaryA = Number(a.getAttribute('data-salary').replace(/[$,]/g, ''));
```

Comment:
Consider checking if 'a.getAttribute('data-salary')' is not null before calling the 'replace' method to prevent possible null reference errors.